### ISSUE TITLE
Create Recommender interface

### ISSUE THREAD

**Issue by frankcleary** [fajifr/recontent#4]
Each recommender will implement a common interface. We need to define this interface in python so everyone can implement it, something like:

``` python
class Recommender:
    getRecommenderId():  # return the ID of this implementation for tracking
        raise NotImplementedError()
    getRecommendation(id):  # return a list of IDs of recommended articles
        raise NotImplementedError()
    getRecommendationFromText(text):  # process text and return list of IDs of recommended 
articles
        raise NotImplementedError()
```

**Comment by Freija** [comment]
A first version was implemented with commit 1e4b374.

**Comment by Freija** [comment]
Done!
